pkg/daemon/update.go

```diff
@@ -490,25 +517,25 @@ func calculatePostConfigChangeActionFromFileDiffs(oldIgnConfig, newIgnConfig ign
 	return
 }
 
-func calculatePostConfigChangeAction(diff *machineConfigDiff, oldIgnConfig, newIgnConfig ign3types.Config) ([]string, error) {
+func calculateConfigChangeActions(diff *machineConfigDiff, oldIgnConfig, newIgnConfig ign3types.Config) (bool, []string, error) {
```

Comment:
why add a bool here instead of just adding drain to the string being returned? returning a bool from a func returning actions seems unexpected.